pandas/io/tests/test_clipboard.py

```diff
@@ -71,6 +71,7 @@ def check_round_trip_frame(self, data_type, excel=None, sep=None):
     def test_round_trip_frame_sep(self):
```

Comment:
This actually just used `read_csv` under the hood, so I don't know to what extent this is needed? (apart from checking that the keyword is passed correctly)